### ISSUE TITLE
Difference in the number of data during deep learning

### ISSUE THREAD

**Issue by rurujisu** [huggingface/transformers#17695]
### System Info

```shell
Python 3
```

### Information

- [ ] The official example scripts
- [ ] My own modified scripts

### Tasks

- [ ] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [ ] My own task or dataset (give details below)

### Reproduction

import matplotlib.pyplot as plt
import matplotlib.tri as tri
import numpy as np
import pandas as pd
from keras.models import Sequential
from keras.layers import Dense, Activation

input = pd.read_csv('D:\\puri\\capture\\3_1\\all\\inlet.csv', sep='\t', skiprows=0)
output = pd.read_csv('D:\\puri\\capture\\3_1\\all\\output.csv', sep='\t', skiprows=0)

model = Sequential([
    Dense(32, input_shape=(784,)),
    Activation('relu'),
    Dense(10),
    Activation('softmax'),
])

model = Sequential()
model.add(Dense(32, input_dim=784))
model.add(Activation('relu'))

### Expected behavior

```shell
hi i just started studying machine learning
I followed the machine learning example, but I can't apply it, so I'm asking here


I want to know the correlation between input data and output data, 
but the number of input data is 500(500 x 7) and the number of output data is 1.8 million(1.8M x 4). In this case, what model should I study?
(inlet variables : 7, output variables : 4)

And i have 5 input-output cases, is that enough to find the correlation?

Thanks for reading let me know if I need to find another way
```

**Comment by gante** [comment]
Hi @rurujisu  👋 As per our [issues guidelines](https://github.com/huggingface/transformers/blob/main/ISSUES.md), we reserve GitHub issues for bugs in the repository and/or feature requests. For any other requests, we'd like to invite you to use our [forum](https://discuss.huggingface.co/) 🤗
